Debian #415585; adapted from a patch found in the Ubuntu package in version
1.8.1-0ubuntu5, itself taken from Fedora.

diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Africa/Cairo.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Africa/Cairo.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Africa/Cairo.ics	2001-10-30 13:45:20.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Africa/Cairo.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Africa/Cairo
+TZID:/softwarestudio.org/Olson_20070227_6/Africa/Cairo
 X-LIC-LOCATION:Africa/Cairo
 BEGIN:DAYLIGHT
 TZOFFSETFROM:+0200
 TZOFFSETTO:+0300
 TZNAME:EEST
 DTSTART:19700424T000000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=-1FR
+RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=-1TH
 END:DAYLIGHT
 BEGIN:STANDARD
-TZOFFSETFROM:+0300
+TZOFFSETFROM:+0200
 TZOFFSETTO:+0200
 TZNAME:EET
-DTSTART:19700925T000000
-RRULE:FREQ=YEARLY;BYMONTH=9;BYDAY=-1FR
+DTSTART:19700924T230000
+RRULE:FREQ=YEARLY;BYMONTH=9;BYDAY=-1TH
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Africa/Tunis.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Africa/Tunis.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Africa/Tunis.ics	2001-10-30 13:45:20.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Africa/Tunis.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,13 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Africa/Tunis
+TZID:/softwarestudio.org/Olson_20070227_6/Africa/Tunis
 X-LIC-LOCATION:Africa/Tunis
-BEGIN:STANDARD
+BEGIN:DAYLIGHT
 TZOFFSETFROM:+0100
+TZOFFSETTO:+0200
+TZNAME:CEST
+DTSTART:19700329T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
+END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:+0200
 TZOFFSETTO:+0100
 TZNAME:CET
-DTSTART:19700101T000000
+DTSTART:19701025T030000
+RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Adak.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Adak.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Adak.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Adak.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Adak
+TZID:/softwarestudio.org/Olson_20070227_6/America/Adak
 X-LIC-LOCATION:America/Adak
-BEGIN:STANDARD
-TZOFFSETFROM:-0900
-TZOFFSETTO:-1000
-TZNAME:HAST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-1000
 TZOFFSETTO:-0900
 TZNAME:HADT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0900
+TZOFFSETTO:-1000
+TZNAME:HAST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Anchorage.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Anchorage.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Anchorage.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Anchorage.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Anchorage
+TZID:/softwarestudio.org/Olson_20070227_6/America/Anchorage
 X-LIC-LOCATION:America/Anchorage
-BEGIN:STANDARD
-TZOFFSETFROM:-0800
-TZOFFSETTO:-0900
-TZNAME:AKST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0900
 TZOFFSETTO:-0800
 TZNAME:AKDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0800
+TZOFFSETTO:-0900
+TZNAME:AKST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Araguaina.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Araguaina.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Araguaina.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Araguaina.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,13 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Araguaina
+TZID:/softwarestudio.org/Olson_20070227_6/America/Araguaina
 X-LIC-LOCATION:America/Araguaina
-BEGIN:DAYLIGHT
-TZOFFSETFROM:-0300
-TZOFFSETTO:-0200
-TZNAME:BRST
-DTSTART:19701011T000000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=2SU
-END:DAYLIGHT
 BEGIN:STANDARD
-TZOFFSETFROM:-0200
+TZOFFSETFROM:-0300
 TZOFFSETTO:-0300
 TZNAME:BRT
-DTSTART:19700215T000000
-RRULE:FREQ=YEARLY;BYMONTH=2;BYDAY=3SU
+DTSTART:19700101T000000
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Asuncion.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Asuncion.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Asuncion.ics	2002-07-26 15:52:39.000000000 +0200
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Asuncion.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20020614_6/America/Asuncion
+TZID:/softwarestudio.org/Olson_20070227_7/America/Asuncion
 X-LIC-LOCATION:America/Asuncion
-BEGIN:STANDARD
-TZOFFSETFROM:-0300
-TZOFFSETTO:-0400
-TZNAME:PYT
-DTSTART:19700405T000000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0400
 TZOFFSETTO:-0300
 TZNAME:PYST
-DTSTART:19700906T000000
-RRULE:FREQ=YEARLY;BYMONTH=9;BYDAY=1SU
+DTSTART:19701018T000000
+RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=3SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0300
+TZOFFSETTO:-0400
+TZNAME:PYT
+DTSTART:19700308T000000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Boise.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Boise.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Boise.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Boise.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Boise
+TZID:/softwarestudio.org/Olson_20070227_6/America/Boise
 X-LIC-LOCATION:America/Boise
-BEGIN:STANDARD
-TZOFFSETFROM:-0600
-TZOFFSETTO:-0700
-TZNAME:MST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0700
 TZOFFSETTO:-0600
 TZNAME:MDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0600
+TZOFFSETTO:-0700
+TZNAME:MST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Cambridge_Bay.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Cambridge_Bay.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Cambridge_Bay.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Cambridge_Bay.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Cambridge_Bay
+TZID:/softwarestudio.org/Olson_20070227_6/America/Cambridge_Bay
 X-LIC-LOCATION:America/Cambridge_Bay
-BEGIN:STANDARD
-TZOFFSETFROM:-0600
-TZOFFSETTO:-0700
-TZNAME:MST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0700
 TZOFFSETTO:-0600
 TZNAME:MDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0600
+TZOFFSETTO:-0700
+TZNAME:MST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Chicago.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Chicago.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Chicago.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Chicago.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Chicago
+TZID:/softwarestudio.org/Olson_20070227_6/America/Chicago
 X-LIC-LOCATION:America/Chicago
-BEGIN:STANDARD
-TZOFFSETFROM:-0500
-TZOFFSETTO:-0600
-TZNAME:CST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0600
 TZOFFSETTO:-0500
 TZNAME:CDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0500
+TZOFFSETTO:-0600
+TZNAME:CST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Cuiaba.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Cuiaba.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Cuiaba.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Cuiaba.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Cuiaba
+TZID:/softwarestudio.org/Olson_20070227_6/America/Cuiaba
 X-LIC-LOCATION:America/Cuiaba
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0400
 TZOFFSETTO:-0300
 TZNAME:AMST
-DTSTART:19701011T000000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=2SU
+DTSTART:19701101T000000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
 END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:-0300
 TZOFFSETTO:-0400
 TZNAME:AMT
-DTSTART:19700215T000000
-RRULE:FREQ=YEARLY;BYMONTH=2;BYDAY=3SU
+DTSTART:19700222T000000
+RRULE:FREQ=YEARLY;BYMONTH=2;BYDAY=-1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Dawson.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Dawson.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Dawson.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Dawson.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Dawson
+TZID:/softwarestudio.org/Olson_20070227_6/America/Dawson
 X-LIC-LOCATION:America/Dawson
-BEGIN:STANDARD
-TZOFFSETFROM:-0700
-TZOFFSETTO:-0800
-TZNAME:PST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0800
 TZOFFSETTO:-0700
 TZNAME:PDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0700
+TZOFFSETTO:-0800
+TZNAME:PST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Denver.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Denver.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Denver.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Denver.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Denver
+TZID:/softwarestudio.org/Olson_20070227_6/America/Denver
 X-LIC-LOCATION:America/Denver
-BEGIN:STANDARD
-TZOFFSETFROM:-0600
-TZOFFSETTO:-0700
-TZNAME:MST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0700
 TZOFFSETTO:-0600
 TZNAME:MDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0600
+TZOFFSETTO:-0700
+TZNAME:MST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Detroit.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Detroit.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Detroit.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Detroit.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Detroit
+TZID:/softwarestudio.org/Olson_20070227_6/America/Detroit
 X-LIC-LOCATION:America/Detroit
-BEGIN:STANDARD
-TZOFFSETFROM:-0400
-TZOFFSETTO:-0500
-TZNAME:EST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0500
 TZOFFSETTO:-0400
 TZNAME:EDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0400
+TZOFFSETTO:-0500
+TZNAME:EST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Edmonton.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Edmonton.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Edmonton.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Edmonton.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Edmonton
+TZID:/softwarestudio.org/Olson_20070227_6/America/Edmonton
 X-LIC-LOCATION:America/Edmonton
-BEGIN:STANDARD
-TZOFFSETFROM:-0600
-TZOFFSETTO:-0700
-TZNAME:MST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0700
 TZOFFSETTO:-0600
 TZNAME:MDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0600
+TZOFFSETTO:-0700
+TZNAME:MST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Fortaleza.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Fortaleza.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Fortaleza.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Fortaleza.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,13 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Fortaleza
+TZID:/softwarestudio.org/Olson_20070227_6/America/Fortaleza
 X-LIC-LOCATION:America/Fortaleza
-BEGIN:DAYLIGHT
-TZOFFSETFROM:-0300
-TZOFFSETTO:-0200
-TZNAME:BRST
-DTSTART:19701011T000000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=2SU
-END:DAYLIGHT
 BEGIN:STANDARD
-TZOFFSETFROM:-0200
+TZOFFSETFROM:-0300
 TZOFFSETTO:-0300
 TZNAME:BRT
-DTSTART:19700215T000000
-RRULE:FREQ=YEARLY;BYMONTH=2;BYDAY=3SU
+DTSTART:19700101T000000
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Glace_Bay.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Glace_Bay.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Glace_Bay.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Glace_Bay.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Glace_Bay
+TZID:/softwarestudio.org/Olson_20070227_6/America/Glace_Bay
 X-LIC-LOCATION:America/Glace_Bay
-BEGIN:STANDARD
-TZOFFSETFROM:-0300
-TZOFFSETTO:-0400
-TZNAME:AST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0400
 TZOFFSETTO:-0300
 TZNAME:ADT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0300
+TZOFFSETTO:-0400
+TZNAME:AST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Goose_Bay.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Goose_Bay.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Goose_Bay.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Goose_Bay.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Goose_Bay
+TZID:/softwarestudio.org/Olson_20070227_6/America/Goose_Bay
 X-LIC-LOCATION:America/Goose_Bay
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0400
 TZOFFSETTO:-0300
 TZNAME:ADT
-DTSTART:19700405T000100
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T000100
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:-0300
 TZOFFSETTO:-0400
 TZNAME:AST
-DTSTART:19701025T000100
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
+DTSTART:19701101T000100
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Halifax.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Halifax.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Halifax.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Halifax.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Halifax
+TZID:/softwarestudio.org/Olson_20070227_6/America/Halifax
 X-LIC-LOCATION:America/Halifax
-BEGIN:STANDARD
-TZOFFSETFROM:-0300
-TZOFFSETTO:-0400
-TZNAME:AST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0400
 TZOFFSETTO:-0300
 TZNAME:ADT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0300
+TZOFFSETTO:-0400
+TZNAME:AST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Indiana/Indianapolis.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Indiana/Indianapolis.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Indiana/Indianapolis.ics	2001-10-30 13:45:23.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Indiana/Indianapolis.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,13 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Indiana/Indianapolis
+TZID:/softwarestudio.org/Olson_20070227_6/America/Indiana/Indianapolis
 X-LIC-LOCATION:America/Indiana/Indianapolis
-BEGIN:STANDARD
+BEGIN:DAYLIGHT
 TZOFFSETFROM:-0500
+TZOFFSETTO:-0400
+TZNAME:EDT
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
+END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0400
 TZOFFSETTO:-0500
 TZNAME:EST
-DTSTART:19700101T000000
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Indiana/Knox.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Indiana/Knox.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Indiana/Knox.ics	2001-10-30 13:45:23.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Indiana/Knox.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,13 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Indiana/Knox
+TZID:/softwarestudio.org/Olson_20070227_6/America/Indiana/Knox
 X-LIC-LOCATION:America/Indiana/Knox
+BEGIN:DAYLIGHT
+TZOFFSETFROM:-0600
+TZOFFSETTO:-0500
+TZNAME:CDT
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
+END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:-0500
-TZOFFSETTO:-0500
-TZNAME:EST
-DTSTART:19700101T000000
+TZOFFSETTO:-0600
+TZNAME:CST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Indiana/Marengo.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Indiana/Marengo.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Indiana/Marengo.ics	2001-10-30 13:45:23.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Indiana/Marengo.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,13 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Indiana/Marengo
+TZID:/softwarestudio.org/Olson_20070227_6/America/Indiana/Marengo
 X-LIC-LOCATION:America/Indiana/Marengo
-BEGIN:STANDARD
+BEGIN:DAYLIGHT
 TZOFFSETFROM:-0500
+TZOFFSETTO:-0400
+TZNAME:EDT
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
+END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0400
 TZOFFSETTO:-0500
 TZNAME:EST
-DTSTART:19700101T000000
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Indiana/Vevay.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Indiana/Vevay.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Indiana/Vevay.ics	2001-10-30 13:45:23.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Indiana/Vevay.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,13 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Indiana/Vevay
+TZID:/softwarestudio.org/Olson_20070227_6/America/Indiana/Vevay
 X-LIC-LOCATION:America/Indiana/Vevay
-BEGIN:STANDARD
+BEGIN:DAYLIGHT
 TZOFFSETFROM:-0500
+TZOFFSETTO:-0400
+TZNAME:EDT
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
+END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0400
 TZOFFSETTO:-0500
 TZNAME:EST
-DTSTART:19700101T000000
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Inuvik.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Inuvik.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Inuvik.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Inuvik.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Inuvik
+TZID:/softwarestudio.org/Olson_20070227_6/America/Inuvik
 X-LIC-LOCATION:America/Inuvik
-BEGIN:STANDARD
-TZOFFSETFROM:-0600
-TZOFFSETTO:-0700
-TZNAME:MST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0700
 TZOFFSETTO:-0600
 TZNAME:MDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0600
+TZOFFSETTO:-0700
+TZNAME:MST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Iqaluit.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Iqaluit.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Iqaluit.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Iqaluit.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Iqaluit
+TZID:/softwarestudio.org/Olson_20070227_6/America/Iqaluit
 X-LIC-LOCATION:America/Iqaluit
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0500
 TZOFFSETTO:-0400
 TZNAME:EDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:-0400
 TZOFFSETTO:-0500
 TZNAME:EST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Juneau.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Juneau.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Juneau.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Juneau.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Juneau
+TZID:/softwarestudio.org/Olson_20070227_6/America/Juneau
 X-LIC-LOCATION:America/Juneau
-BEGIN:STANDARD
-TZOFFSETFROM:-0800
-TZOFFSETTO:-0900
-TZNAME:AKST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0900
 TZOFFSETTO:-0800
 TZNAME:AKDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0800
+TZOFFSETTO:-0900
+TZNAME:AKST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Kentucky/Louisville.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Kentucky/Louisville.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Kentucky/Louisville.ics	2001-10-30 13:45:24.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Kentucky/Louisville.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Kentucky/Louisville
+TZID:/softwarestudio.org/Olson_20070227_6/America/Kentucky/Louisville
 X-LIC-LOCATION:America/Kentucky/Louisville
-BEGIN:STANDARD
-TZOFFSETFROM:-0400
-TZOFFSETTO:-0500
-TZNAME:EST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0500
 TZOFFSETTO:-0400
 TZNAME:EDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0400
+TZOFFSETTO:-0500
+TZNAME:EST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Kentucky/Monticello.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Kentucky/Monticello.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Kentucky/Monticello.ics	2001-10-30 13:45:24.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Kentucky/Monticello.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Kentucky/Monticello
+TZID:/softwarestudio.org/Olson_20070227_6/America/Kentucky/Monticello
 X-LIC-LOCATION:America/Kentucky/Monticello
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0500
 TZOFFSETTO:-0400
 TZNAME:EDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:-0400
 TZOFFSETTO:-0500
 TZNAME:EST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Los_Angeles.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Los_Angeles.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Los_Angeles.ics	2006-08-28 20:19:31.000000000 +0200
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Los_Angeles.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,51 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20050423_1/America/Los_Angeles
+TZID:/softwarestudio.org/Olson_20070227_2/America/Los_Angeles
 X-LIC-LOCATION:America/Los_Angeles
-BEGIN:STANDARD
-TZOFFSETFROM:-0700
-TZOFFSETTO:-0800
-TZNAME:PST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0800
 TZOFFSETTO:-0700
 TZNAME:PDT
-DTSTART:19870405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
 BEGIN:STANDARD
-TZOFFSETFROM:-0800
+TZOFFSETFROM:-0700
 TZOFFSETTO:-0800
 TZNAME:PST
-DTSTART:19700101T000000
-RDATE:19700101T000000
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
 END:STANDARD
-BEGIN:DAYLIGHT
-TZOFFSETFROM:-0800
-TZOFFSETTO:-0700
-TZNAME:PDT
-DTSTART:19700426T020000
-RDATE:19700426T020000
-RDATE:19710425T020000
-RDATE:19720430T020000
-RDATE:19730429T020000
-RDATE:19740106T020000
-RDATE:19750223T020000
-RDATE:19760425T020000
-RDATE:19770424T020000
-RDATE:19780430T020000
-RDATE:19790429T020000
-RDATE:19800427T020000
-RDATE:19810426T020000
-RDATE:19820425T020000
-RDATE:19830424T020000
-RDATE:19840429T020000
-RDATE:19850428T020000
-RDATE:19860427T020000
-END:DAYLIGHT
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Maceio.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Maceio.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Maceio.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Maceio.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,13 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Maceio
+TZID:/softwarestudio.org/Olson_20070227_6/America/Maceio
 X-LIC-LOCATION:America/Maceio
-BEGIN:DAYLIGHT
-TZOFFSETFROM:-0300
-TZOFFSETTO:-0200
-TZNAME:BRST
-DTSTART:19701011T000000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=2SU
-END:DAYLIGHT
 BEGIN:STANDARD
-TZOFFSETFROM:-0200
+TZOFFSETFROM:-0300
 TZOFFSETTO:-0300
 TZNAME:BRT
-DTSTART:19700215T000000
-RRULE:FREQ=YEARLY;BYMONTH=2;BYDAY=3SU
+DTSTART:19700101T000000
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Menominee.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Menominee.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Menominee.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Menominee.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Menominee
+TZID:/softwarestudio.org/Olson_20070227_6/America/Menominee
 X-LIC-LOCATION:America/Menominee
-BEGIN:STANDARD
-TZOFFSETFROM:-0500
-TZOFFSETTO:-0600
-TZNAME:CST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0600
 TZOFFSETTO:-0500
 TZNAME:CDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0500
+TZOFFSETTO:-0600
+TZNAME:CST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Miquelon.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Miquelon.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Miquelon.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Miquelon.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Miquelon
+TZID:/softwarestudio.org/Olson_20070227_6/America/Miquelon
 X-LIC-LOCATION:America/Miquelon
-BEGIN:STANDARD
-TZOFFSETFROM:-0200
-TZOFFSETTO:-0300
-TZNAME:PMST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0300
 TZOFFSETTO:-0200
 TZNAME:PMDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0200
+TZOFFSETTO:-0300
+TZNAME:PMST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Montevideo.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Montevideo.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Montevideo.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Montevideo.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,13 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Montevideo
+TZID:/softwarestudio.org/Olson_20070227_6/America/Montevideo
 X-LIC-LOCATION:America/Montevideo
-BEGIN:STANDARD
+BEGIN:DAYLIGHT
 TZOFFSETFROM:-0300
+TZOFFSETTO:-0200
+TZNAME:UYST
+DTSTART:19701004T020000
+RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=1SU
+END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0200
 TZOFFSETTO:-0300
 TZNAME:UYT
-DTSTART:19700101T000000
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Montreal.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Montreal.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Montreal.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Montreal.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Montreal
+TZID:/softwarestudio.org/Olson_20070227_6/America/Montreal
 X-LIC-LOCATION:America/Montreal
-BEGIN:STANDARD
-TZOFFSETFROM:-0400
-TZOFFSETTO:-0500
-TZNAME:EST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0500
 TZOFFSETTO:-0400
 TZNAME:EDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0400
+TZOFFSETTO:-0500
+TZNAME:EST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Nassau.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Nassau.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Nassau.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Nassau.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Nassau
+TZID:/softwarestudio.org/Olson_20070227_6/America/Nassau
 X-LIC-LOCATION:America/Nassau
-BEGIN:STANDARD
-TZOFFSETFROM:-0400
-TZOFFSETTO:-0500
-TZNAME:EST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0500
 TZOFFSETTO:-0400
 TZNAME:EDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0400
+TZOFFSETTO:-0500
+TZNAME:EST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/New_York.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/New_York.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/New_York.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/New_York.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/New_York
+TZID:/softwarestudio.org/Olson_20070227_6/America/New_York
 X-LIC-LOCATION:America/New_York
-BEGIN:STANDARD
-TZOFFSETFROM:-0400
-TZOFFSETTO:-0500
-TZNAME:EST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0500
 TZOFFSETTO:-0400
 TZNAME:EDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0400
+TZOFFSETTO:-0500
+TZNAME:EST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Nipigon.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Nipigon.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Nipigon.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Nipigon.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Nipigon
+TZID:/softwarestudio.org/Olson_20070227_6/America/Nipigon
 X-LIC-LOCATION:America/Nipigon
-BEGIN:STANDARD
-TZOFFSETFROM:-0400
-TZOFFSETTO:-0500
-TZNAME:EST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0500
 TZOFFSETTO:-0400
 TZNAME:EDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0400
+TZOFFSETTO:-0500
+TZNAME:EST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Nome.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Nome.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Nome.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Nome.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Nome
+TZID:/softwarestudio.org/Olson_20070227_6/America/Nome
 X-LIC-LOCATION:America/Nome
-BEGIN:STANDARD
-TZOFFSETFROM:-0800
-TZOFFSETTO:-0900
-TZNAME:AKST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0900
 TZOFFSETTO:-0800
 TZNAME:AKDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0800
+TZOFFSETTO:-0900
+TZNAME:AKST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/North_Dakota/Center.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/North_Dakota/Center.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/North_Dakota/Center.ics	2001-10-30 13:45:24.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/North_Dakota/Center.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_2/America/North_Dakota/Center
+TZID:/softwarestudio.org/Olson_20070227_3/America/North_Dakota/Center
 X-LIC-LOCATION:America/North_Dakota/Center
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0600
 TZOFFSETTO:-0500
 TZNAME:CDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:-0500
 TZOFFSETTO:-0600
 TZNAME:CST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Pangnirtung.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Pangnirtung.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Pangnirtung.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Pangnirtung.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Pangnirtung
+TZID:/softwarestudio.org/Olson_20070227_6/America/Pangnirtung
 X-LIC-LOCATION:America/Pangnirtung
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0500
 TZOFFSETTO:-0400
 TZNAME:EDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:-0400
 TZOFFSETTO:-0500
 TZNAME:EST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Port-au-Prince.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Port-au-Prince.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Port-au-Prince.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Port-au-Prince.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,13 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Port-au-Prince
+TZID:/softwarestudio.org/Olson_20070227_6/America/Port-au-Prince
 X-LIC-LOCATION:America/Port-au-Prince
-BEGIN:STANDARD
+BEGIN:DAYLIGHT
 TZOFFSETFROM:-0500
+TZOFFSETTO:-0400
+TZNAME:EDT
+DTSTART:19700405T000000
+RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0400
 TZOFFSETTO:-0500
 TZNAME:EST
-DTSTART:19700101T000000
+DTSTART:19701025T000000
+RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Rainy_River.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Rainy_River.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Rainy_River.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Rainy_River.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Rainy_River
+TZID:/softwarestudio.org/Olson_20070227_6/America/Rainy_River
 X-LIC-LOCATION:America/Rainy_River
-BEGIN:STANDARD
-TZOFFSETFROM:-0500
-TZOFFSETTO:-0600
-TZNAME:CST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0600
 TZOFFSETTO:-0500
 TZNAME:CDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0500
+TZOFFSETTO:-0600
+TZNAME:CST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Rankin_Inlet.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Rankin_Inlet.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Rankin_Inlet.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Rankin_Inlet.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Rankin_Inlet
+TZID:/softwarestudio.org/Olson_20070227_6/America/Rankin_Inlet
 X-LIC-LOCATION:America/Rankin_Inlet
-BEGIN:STANDARD
-TZOFFSETFROM:-0500
-TZOFFSETTO:-0600
-TZNAME:CST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0600
 TZOFFSETTO:-0500
 TZNAME:CDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0500
+TZOFFSETTO:-0600
+TZNAME:CST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Recife.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Recife.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Recife.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Recife.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,13 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Recife
+TZID:/softwarestudio.org/Olson_20070227_6/America/Recife
 X-LIC-LOCATION:America/Recife
-BEGIN:DAYLIGHT
-TZOFFSETFROM:-0300
-TZOFFSETTO:-0200
-TZNAME:BRST
-DTSTART:19701011T000000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=2SU
-END:DAYLIGHT
 BEGIN:STANDARD
-TZOFFSETFROM:-0200
+TZOFFSETFROM:-0300
 TZOFFSETTO:-0300
 TZNAME:BRT
-DTSTART:19700215T000000
-RRULE:FREQ=YEARLY;BYMONTH=2;BYDAY=3SU
+DTSTART:19700101T000000
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Sao_Paulo.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Sao_Paulo.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Sao_Paulo.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Sao_Paulo.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Sao_Paulo
+TZID:/softwarestudio.org/Olson_20070227_6/America/Sao_Paulo
 X-LIC-LOCATION:America/Sao_Paulo
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0300
 TZOFFSETTO:-0200
 TZNAME:BRST
-DTSTART:19701011T000000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=2SU
+DTSTART:19701101T000000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
 END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:-0200
 TZOFFSETTO:-0300
 TZNAME:BRT
-DTSTART:19700215T000000
-RRULE:FREQ=YEARLY;BYMONTH=2;BYDAY=3SU
+DTSTART:19700222T000000
+RRULE:FREQ=YEARLY;BYMONTH=2;BYDAY=-1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Shiprock.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Shiprock.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Shiprock.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Shiprock.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Shiprock
+TZID:/softwarestudio.org/Olson_20070227_6/America/Shiprock
 X-LIC-LOCATION:America/Shiprock
-BEGIN:STANDARD
-TZOFFSETFROM:-0600
-TZOFFSETTO:-0700
-TZNAME:MST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0700
 TZOFFSETTO:-0600
 TZNAME:MDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0600
+TZOFFSETTO:-0700
+TZNAME:MST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/St_Johns.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/St_Johns.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/St_Johns.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/St_Johns.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/St_Johns
+TZID:/softwarestudio.org/Olson_20070227_6/America/St_Johns
 X-LIC-LOCATION:America/St_Johns
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0330
 TZOFFSETTO:-0230
 TZNAME:NDT
-DTSTART:19700405T000100
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T000100
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:-0230
 TZOFFSETTO:-0330
 TZNAME:NST
-DTSTART:19701025T000100
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
+DTSTART:19701101T000100
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Thule.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Thule.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Thule.ics	2002-07-26 15:52:39.000000000 +0200
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Thule.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20020614_6/America/Thule
+TZID:/softwarestudio.org/Olson_20070227_7/America/Thule
 X-LIC-LOCATION:America/Thule
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0400
 TZOFFSETTO:-0300
 TZNAME:ADT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:-0300
 TZOFFSETTO:-0400
 TZNAME:AST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Thunder_Bay.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Thunder_Bay.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Thunder_Bay.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Thunder_Bay.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Thunder_Bay
+TZID:/softwarestudio.org/Olson_20070227_6/America/Thunder_Bay
 X-LIC-LOCATION:America/Thunder_Bay
-BEGIN:STANDARD
-TZOFFSETFROM:-0400
-TZOFFSETTO:-0500
-TZNAME:EST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0500
 TZOFFSETTO:-0400
 TZNAME:EDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0400
+TZOFFSETTO:-0500
+TZNAME:EST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Vancouver.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Vancouver.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Vancouver.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Vancouver.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Vancouver
+TZID:/softwarestudio.org/Olson_20070227_6/America/Vancouver
 X-LIC-LOCATION:America/Vancouver
-BEGIN:STANDARD
-TZOFFSETFROM:-0700
-TZOFFSETTO:-0800
-TZNAME:PST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0800
 TZOFFSETTO:-0700
 TZNAME:PDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0700
+TZOFFSETTO:-0800
+TZNAME:PST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Whitehorse.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Whitehorse.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Whitehorse.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Whitehorse.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Whitehorse
+TZID:/softwarestudio.org/Olson_20070227_6/America/Whitehorse
 X-LIC-LOCATION:America/Whitehorse
-BEGIN:STANDARD
-TZOFFSETFROM:-0700
-TZOFFSETTO:-0800
-TZNAME:PST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0800
 TZOFFSETTO:-0700
 TZNAME:PDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0700
+TZOFFSETTO:-0800
+TZNAME:PST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Winnipeg.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Winnipeg.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Winnipeg.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Winnipeg.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Winnipeg
+TZID:/softwarestudio.org/Olson_20070227_6/America/Winnipeg
 X-LIC-LOCATION:America/Winnipeg
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0600
 TZOFFSETTO:-0500
 TZNAME:CDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:-0500
 TZOFFSETTO:-0600
 TZNAME:CST
-DTSTART:19701025T030000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Yakutat.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Yakutat.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Yakutat.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Yakutat.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Yakutat
+TZID:/softwarestudio.org/Olson_20070227_6/America/Yakutat
 X-LIC-LOCATION:America/Yakutat
-BEGIN:STANDARD
-TZOFFSETFROM:-0800
-TZOFFSETTO:-0900
-TZNAME:AKST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0900
 TZOFFSETTO:-0800
 TZNAME:AKDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0800
+TZOFFSETTO:-0900
+TZNAME:AKST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Yellowknife.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Yellowknife.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/America/Yellowknife.ics	2001-10-30 13:45:22.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/America/Yellowknife.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/America/Yellowknife
+TZID:/softwarestudio.org/Olson_20070227_6/America/Yellowknife
 X-LIC-LOCATION:America/Yellowknife
-BEGIN:STANDARD
-TZOFFSETFROM:-0600
-TZOFFSETTO:-0700
-TZNAME:MST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0700
 TZOFFSETTO:-0600
 TZNAME:MDT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0600
+TZOFFSETTO:-0700
+TZNAME:MST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Almaty.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Almaty.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Almaty.ics	2002-07-26 15:52:40.000000000 +0200
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Almaty.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,13 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20020614_6/Asia/Almaty
+TZID:/softwarestudio.org/Olson_20070227_7/Asia/Almaty
 X-LIC-LOCATION:Asia/Almaty
-BEGIN:DAYLIGHT
-TZOFFSETFROM:+0600
-TZOFFSETTO:+0700
-TZNAME:ALMST
-DTSTART:19700329T020000
-RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
-END:DAYLIGHT
 BEGIN:STANDARD
-TZOFFSETFROM:+0700
+TZOFFSETFROM:+0600
 TZOFFSETTO:+0600
 TZNAME:ALMT
-DTSTART:19701025T030000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
+DTSTART:19700101T000000
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Amman.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Amman.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Amman.ics	2001-10-30 13:45:26.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Amman.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,15 +2,8 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Asia/Amman
+TZID:/softwarestudio.org/Olson_20070227_6/Asia/Amman
 X-LIC-LOCATION:Asia/Amman
-BEGIN:STANDARD
-TZOFFSETFROM:+0300
-TZOFFSETTO:+0200
-TZNAME:EET
-DTSTART:19700924T010000
-RRULE:FREQ=YEARLY;BYMONTH=9;BYDAY=-1TH
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:+0200
 TZOFFSETTO:+0300
@@ -18,5 +11,12 @@
 DTSTART:19700326T000000
 RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1TH
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:+0300
+TZOFFSETTO:+0200
+TZNAME:EET
+DTSTART:19701030T010000
+RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1FR
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Aqtau.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Aqtau.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Aqtau.ics	2002-07-26 15:52:40.000000000 +0200
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Aqtau.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,13 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20020614_6/Asia/Aqtau
+TZID:/softwarestudio.org/Olson_20070227_7/Asia/Aqtau
 X-LIC-LOCATION:Asia/Aqtau
-BEGIN:DAYLIGHT
-TZOFFSETFROM:+0400
-TZOFFSETTO:+0500
-TZNAME:AQTST
-DTSTART:19700329T020000
-RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
-END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:+0500
-TZOFFSETTO:+0400
+TZOFFSETTO:+0500
 TZNAME:AQTT
-DTSTART:19701025T030000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
+DTSTART:19700101T000000
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Aqtobe.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Aqtobe.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Aqtobe.ics	2002-07-26 15:52:40.000000000 +0200
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Aqtobe.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,13 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20020614_6/Asia/Aqtobe
+TZID:/softwarestudio.org/Olson_20070227_7/Asia/Aqtobe
 X-LIC-LOCATION:Asia/Aqtobe
-BEGIN:DAYLIGHT
-TZOFFSETFROM:+0500
-TZOFFSETTO:+0600
-TZNAME:AQTST
-DTSTART:19700329T020000
-RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
-END:DAYLIGHT
 BEGIN:STANDARD
-TZOFFSETFROM:+0600
+TZOFFSETFROM:+0500
 TZOFFSETTO:+0500
 TZNAME:AQTT
-DTSTART:19701025T030000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
+DTSTART:19700101T000000
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Baku.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Baku.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Baku.ics	2001-10-30 13:45:26.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Baku.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,20 +2,20 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Asia/Baku
+TZID:/softwarestudio.org/Olson_20070227_6/Asia/Baku
 X-LIC-LOCATION:Asia/Baku
 BEGIN:DAYLIGHT
 TZOFFSETFROM:+0400
 TZOFFSETTO:+0500
 TZNAME:AZST
-DTSTART:19700329T010000
+DTSTART:19700329T040000
 RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
 END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:+0500
 TZOFFSETTO:+0400
 TZNAME:AZT
-DTSTART:19701025T010000
+DTSTART:19701025T050000
 RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
 END:STANDARD
 END:VTIMEZONE
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Bishkek.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Bishkek.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Bishkek.ics	2001-10-30 13:45:26.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Bishkek.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,13 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Asia/Bishkek
+TZID:/softwarestudio.org/Olson_20070227_6/Asia/Bishkek
 X-LIC-LOCATION:Asia/Bishkek
-BEGIN:DAYLIGHT
-TZOFFSETFROM:+0500
-TZOFFSETTO:+0600
-TZNAME:KGST
-DTSTART:19700329T023000
-RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
-END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:+0600
-TZOFFSETTO:+0500
+TZOFFSETTO:+0600
 TZNAME:KGT
-DTSTART:19701025T023000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
+DTSTART:19700101T000000
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Choibalsan.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Choibalsan.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Choibalsan.ics	2006-08-07 19:00:24.000000000 +0200
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Choibalsan.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_2/Asia/Choibalsan
+TZID:/softwarestudio.org/Olson_20070227_3/Asia/Choibalsan
 X-LIC-LOCATION:Asia/Choibalsan
-BEGIN:DAYLIGHT
-TZOFFSETFROM:+0800
+BEGIN:STANDARD
+TZOFFSETFROM:+1000
 TZOFFSETTO:+0900
 TZNAME:CHOT
+DTSTART:19700926T020000
+RRULE:FREQ=YEARLY;BYMONTH=9;BYDAY=-1SA
+END:STANDARD
+BEGIN:DAYLIGHT
+TZOFFSETFROM:+0900
+TZOFFSETTO:+1000
+TZNAME:CHOST
 DTSTART:19700328T020000
 RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SA
 END:DAYLIGHT
-BEGIN:STANDARD
-TZOFFSETFROM:+0900
-TZOFFSETTO:+0800
-TZNAME:CHOT
-DTSTART:19701024T030000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SA
-END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Colombo.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Colombo.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Colombo.ics	2006-06-15 14:24:54.000000000 +0200
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Colombo.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,12 +2,12 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Asia/Colombo
+TZID:/softwarestudio.org/Olson_20070227_6/Asia/Colombo
 X-LIC-LOCATION:Asia/Colombo
 BEGIN:STANDARD
 TZOFFSETFROM:+0530
 TZOFFSETTO:+0530
-TZNAME:LKT
+TZNAME:IST
 DTSTART:19700101T000000
 END:STANDARD
 END:VTIMEZONE
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Damascus.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Damascus.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Damascus.ics	2001-10-30 13:45:26.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Damascus.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,15 +2,8 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Asia/Damascus
+TZID:/softwarestudio.org/Olson_20070227_6/Asia/Damascus
 X-LIC-LOCATION:Asia/Damascus
-BEGIN:STANDARD
-TZOFFSETFROM:+0300
-TZOFFSETTO:+0200
-TZNAME:EET
-DTSTART:19701001T000000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:+0200
 TZOFFSETTO:+0300
@@ -18,5 +11,12 @@
 DTSTART:19700401T000000
 RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:+0300
+TZOFFSETTO:+0200
+TZNAME:EET
+DTSTART:19701001T000000
+RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Dili.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Dili.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Dili.ics	2001-10-30 13:45:26.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Dili.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,12 +2,12 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Asia/Dili
+TZID:/softwarestudio.org/Olson_20070227_6/Asia/Dili
 X-LIC-LOCATION:Asia/Dili
 BEGIN:STANDARD
 TZOFFSETFROM:+0900
 TZOFFSETTO:+0900
-TZNAME:TPT
+TZNAME:TLT
 DTSTART:19700101T000000
 END:STANDARD
 END:VTIMEZONE
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Gaza.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Gaza.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Gaza.ics	2001-10-30 13:45:26.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Gaza.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,14 +2,14 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Asia/Gaza
+TZID:/softwarestudio.org/Olson_20070227_6/Asia/Gaza
 X-LIC-LOCATION:Asia/Gaza
 BEGIN:DAYLIGHT
 TZOFFSETFROM:+0200
 TZOFFSETTO:+0300
 TZNAME:EEST
-DTSTART:19700417T000000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=3FR
+DTSTART:19700401T000000
+RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
 END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:+0300
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Hovd.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Hovd.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Hovd.ics	2006-08-07 19:00:24.000000000 +0200
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Hovd.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,13 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Asia/Hovd
+TZID:/softwarestudio.org/Olson_20070227_6/Asia/Hovd
 X-LIC-LOCATION:Asia/Hovd
-BEGIN:DAYLIGHT
-TZOFFSETFROM:+0800
-TZOFFSETTO:+0700
-TZNAME:HOVT
-DTSTART:19700328T020000
-RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SA
-END:DAYLIGHT
 BEGIN:STANDARD
-TZOFFSETFROM:+0800
+TZOFFSETFROM:+0700
 TZOFFSETTO:+0700
 TZNAME:HOVT
-DTSTART:19701024T030000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
+DTSTART:19700101T000000
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Istanbul.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Istanbul.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Istanbul.ics	2001-10-30 13:45:26.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Istanbul.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,20 +2,20 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Asia/Istanbul
+TZID:/softwarestudio.org/Olson_20070227_6/Asia/Istanbul
 X-LIC-LOCATION:Asia/Istanbul
 BEGIN:DAYLIGHT
 TZOFFSETFROM:+0200
 TZOFFSETTO:+0300
 TZNAME:EEST
-DTSTART:19700329T030000
+DTSTART:19700329T010000
 RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
 END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:+0300
 TZOFFSETTO:+0200
 TZNAME:EET
-DTSTART:19701025T040000
+DTSTART:19701025T020000
 RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
 END:STANDARD
 END:VTIMEZONE
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Jerusalem.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Jerusalem.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Jerusalem.ics	2006-09-21 12:22:12.000000000 +0200
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Jerusalem.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,136 +2,19 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20050423_1/Asia/Jerusalem
+TZID:/softwarestudio.org/Olson_20070227_2/Asia/Jerusalem
 X-LIC-LOCATION:Asia/Jerusalem
 BEGIN:STANDARD
-TZOFFSETFROM:+0200
+DTSTART:16010101T020000
+TZOFFSETFROM:+0300
 TZOFFSETTO:+0200
-TZNAME:IST
-DTSTART:19700101T000000
-RDATE:19700101T000000
+RRULE:FREQ=YEARLY;INTERVAL=1;BYDAY=3SU;BYMONTH=9
 END:STANDARD
 BEGIN:DAYLIGHT
+DTSTART:16010101T020000
 TZOFFSETFROM:+0200
 TZOFFSETTO:+0300
-TZNAME:IDT
-DTSTART:19740707T000000
-RDATE:19740707T000000
-RDATE:19750420T000000
-RDATE:19850414T000000
-RDATE:19860518T000000
-RDATE:19870415T000000
-RDATE:19880409T000000
-RDATE:19890430T000000
-RDATE:19900325T000000
-RDATE:19910324T000000
-RDATE:19920329T000000
-RDATE:19930402T000000
-RDATE:19940401T000000
-RDATE:19950331T000000
-RDATE:19960315T000000
-RDATE:19970321T000000
-RDATE:19980320T000000
-RDATE:19990402T020000
-RDATE:20000414T020000
-RDATE:20010409T010000
-RDATE:20020329T010000
-RDATE:20030328T010000
-RDATE:20040407T010000
-RDATE:20050401T020000
-RDATE:20060331T020000
-RDATE:20070330T020000
-RDATE:20080328T020000
-RDATE:20090327T020000
-RDATE:20100326T020000
-RDATE:20110401T020000
-RDATE:20120330T020000
-RDATE:20130329T020000
-RDATE:20140328T020000
-RDATE:20150327T020000
-RDATE:20160401T020000
-RDATE:20170331T020000
-RDATE:20180330T020000
-RDATE:20190329T020000
-RDATE:20200327T020000
-RDATE:20210326T020000
-RDATE:20220401T020000
-RDATE:20230331T020000
-RDATE:20240329T020000
-RDATE:20250328T020000
-RDATE:20260327T020000
-RDATE:20270326T020000
-RDATE:20280331T020000
-RDATE:20290330T020000
-RDATE:20300329T020000
-RDATE:20310328T020000
-RDATE:20320326T020000
-RDATE:20330401T020000
-RDATE:20340331T020000
-RDATE:20350330T020000
-RDATE:20360328T020000
-RDATE:20370327T020000
+RRULE:FREQ=YEARLY;INTERVAL=1;BYDAY=-1FR;BYMONTH=3
 END:DAYLIGHT
-BEGIN:STANDARD
-TZOFFSETFROM:+0300
-TZOFFSETTO:+0200
-TZNAME:IST
-DTSTART:19741013T000000
-RDATE:19741013T000000
-RDATE:19750831T000000
-RDATE:19850915T000000
-RDATE:19860907T000000
-RDATE:19870913T000000
-RDATE:19880903T000000
-RDATE:19890903T000000
-RDATE:19900826T000000
-RDATE:19910901T000000
-RDATE:19920906T000000
-RDATE:19930905T000000
-RDATE:19940828T000000
-RDATE:19950903T000000
-RDATE:19960916T000000
-RDATE:19970914T000000
-RDATE:19980906T000000
-RDATE:19990903T020000
-RDATE:20001006T010000
-RDATE:20010924T010000
-RDATE:20021007T010000
-RDATE:20031003T010000
-RDATE:20040922T010000
-RDATE:20051009T020000
-RDATE:20061001T020000
-RDATE:20070916T020000
-RDATE:20081005T020000
-RDATE:20090927T020000
-RDATE:20100912T020000
-RDATE:20111002T020000
-RDATE:20120923T020000
-RDATE:20130908T020000
-RDATE:20140928T020000
-RDATE:20150920T020000
-RDATE:20161009T020000
-RDATE:20170924T020000
-RDATE:20180916T020000
-RDATE:20191006T020000
-RDATE:20200927T020000
-RDATE:20210912T020000
-RDATE:20221002T020000
-RDATE:20230924T020000
-RDATE:20241006T020000
-RDATE:20250928T020000
-RDATE:20260920T020000
-RDATE:20271010T020000
-RDATE:20280924T020000
-RDATE:20290916T020000
-RDATE:20301006T020000
-RDATE:20310921T020000
-RDATE:20320912T020000
-RDATE:20331002T020000
-RDATE:20340917T020000
-RDATE:20351007T020000
-RDATE:20360928T020000
-RDATE:20370913T020000
-END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Karachi.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Karachi.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Karachi.ics	2002-07-26 15:52:40.000000000 +0200
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Karachi.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,13 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20020614_6/Asia/Karachi
+TZID:/softwarestudio.org/Olson_20070227_7/Asia/Karachi
 X-LIC-LOCATION:Asia/Karachi
-BEGIN:DAYLIGHT
-TZOFFSETFROM:+0500
-TZOFFSETTO:+0600
-TZNAME:PKST
-DTSTART:19700405T000000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
-END:DAYLIGHT
 BEGIN:STANDARD
-TZOFFSETFROM:+0600
+TZOFFSETFROM:+0500
 TZOFFSETTO:+0500
 TZNAME:PKT
-DTSTART:19701015T000000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=3SU
+DTSTART:19700101T000000
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Tbilisi.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Tbilisi.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Tbilisi.ics	2001-10-30 13:45:26.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Tbilisi.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,13 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Asia/Tbilisi
+TZID:/softwarestudio.org/Olson_20070227_6/Asia/Tbilisi
 X-LIC-LOCATION:Asia/Tbilisi
 BEGIN:STANDARD
-TZOFFSETFROM:+0500
+TZOFFSETFROM:+0400
 TZOFFSETTO:+0400
 TZNAME:GET
-DTSTART:19701025T000000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
+DTSTART:19700101T000000
 END:STANDARD
-BEGIN:DAYLIGHT
-TZOFFSETFROM:+0400
-TZOFFSETTO:+0500
-TZNAME:GEST
-DTSTART:19700329T000000
-RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
-END:DAYLIGHT
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Tehran.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Tehran.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Tehran.ics	2001-10-30 13:45:26.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Tehran.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,12 +2,12 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Asia/Tehran
+TZID:/softwarestudio.org/Olson_20070227_6/Asia/Tehran
 X-LIC-LOCATION:Asia/Tehran
 BEGIN:STANDARD
 TZOFFSETFROM:+0330
 TZOFFSETTO:+0330
-TZNAME:IRT
+TZNAME:IRST
 DTSTART:19700101T000000
 END:STANDARD
 END:VTIMEZONE
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Ulaanbaatar.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Ulaanbaatar.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Asia/Ulaanbaatar.ics	2006-08-07 19:00:24.000000000 +0200
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Asia/Ulaanbaatar.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Asia/Ulaanbaatar
+TZID:/softwarestudio.org/Olson_20070227_6/Asia/Ulaanbaatar
 X-LIC-LOCATION:Asia/Ulaanbaatar
+BEGIN:STANDARD
+TZOFFSETFROM:+0900
+TZOFFSETTO:+0800
+TZNAME:ULAT
+DTSTART:19700926T020000
+RRULE:FREQ=YEARLY;BYMONTH=9;BYDAY=-1SA
+END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:+0800
 TZOFFSETTO:+0900
-TZNAME:ULAT
+TZNAME:ULAST
 DTSTART:19700328T020000
 RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SA
 END:DAYLIGHT
-BEGIN:STANDARD
-TZOFFSETFROM:+0900
-TZOFFSETTO:+0800
-TZNAME:ULAT
-DTSTART:19701024T030000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SA
-END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Atlantic/Bermuda.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Atlantic/Bermuda.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Atlantic/Bermuda.ics	2001-10-30 13:45:27.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Atlantic/Bermuda.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Atlantic/Bermuda
+TZID:/softwarestudio.org/Olson_20070227_6/Atlantic/Bermuda
 X-LIC-LOCATION:Atlantic/Bermuda
-BEGIN:STANDARD
-TZOFFSETFROM:-0300
-TZOFFSETTO:-0400
-TZNAME:AST
-DTSTART:19701025T020000
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:-0400
 TZOFFSETTO:-0300
 TZNAME:ADT
-DTSTART:19700405T020000
-RRULE:FREQ=YEARLY;BYMONTH=4;BYDAY=1SU
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0300
+TZOFFSETTO:-0400
+TZNAME:AST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Australia/Adelaide.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Australia/Adelaide.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Australia/Adelaide.ics	2001-10-30 13:45:27.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Australia/Adelaide.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,7 +2,7 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Australia/Adelaide
+TZID:/softwarestudio.org/Olson_20070227_6/Australia/Adelaide
 X-LIC-LOCATION:Australia/Adelaide
 BEGIN:DAYLIGHT
 TZOFFSETFROM:+0930
@@ -12,10 +12,10 @@
 RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
 END:DAYLIGHT
 BEGIN:STANDARD
-TZOFFSETFROM:+1030
+TZOFFSETFROM:+0930
 TZOFFSETTO:+0930
 TZNAME:CST
-DTSTART:19700329T030000
+DTSTART:19700329T020000
 RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
 END:STANDARD
 END:VTIMEZONE
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Australia/Broken_Hill.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Australia/Broken_Hill.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Australia/Broken_Hill.ics	2001-10-30 13:45:27.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Australia/Broken_Hill.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,7 +2,7 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Australia/Broken_Hill
+TZID:/softwarestudio.org/Olson_20070227_6/Australia/Broken_Hill
 X-LIC-LOCATION:Australia/Broken_Hill
 BEGIN:DAYLIGHT
 TZOFFSETFROM:+0930
@@ -12,10 +12,10 @@
 RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
 END:DAYLIGHT
 BEGIN:STANDARD
-TZOFFSETFROM:+1030
+TZOFFSETFROM:+0930
 TZOFFSETTO:+0930
 TZNAME:CST
-DTSTART:19700329T030000
+DTSTART:19700329T020000
 RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
 END:STANDARD
 END:VTIMEZONE
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Australia/Hobart.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Australia/Hobart.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Australia/Hobart.ics	2001-10-30 13:45:27.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Australia/Hobart.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,15 +2,8 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Australia/Hobart
+TZID:/softwarestudio.org/Olson_20070227_6/Australia/Hobart
 X-LIC-LOCATION:Australia/Hobart
-BEGIN:STANDARD
-TZOFFSETFROM:+1100
-TZOFFSETTO:+1000
-TZNAME:EST
-DTSTART:19700329T030000
-RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:+1000
 TZOFFSETTO:+1100
@@ -18,5 +11,12 @@
 DTSTART:19701004T020000
 RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=1SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:+1000
+TZOFFSETTO:+1000
+TZNAME:EST
+DTSTART:19700329T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Australia/Lord_Howe.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Australia/Lord_Howe.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Australia/Lord_Howe.ics	2001-10-30 13:45:27.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Australia/Lord_Howe.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,15 +2,8 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Australia/Lord_Howe
+TZID:/softwarestudio.org/Olson_20070227_6/Australia/Lord_Howe
 X-LIC-LOCATION:Australia/Lord_Howe
-BEGIN:STANDARD
-TZOFFSETFROM:+1100
-TZOFFSETTO:+1030
-TZNAME:LHST
-DTSTART:19700329T020000
-RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:+1030
 TZOFFSETTO:+1100
@@ -18,5 +11,12 @@
 DTSTART:19701025T020000
 RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:+1030
+TZOFFSETTO:+1030
+TZNAME:LHST
+DTSTART:19700329T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Australia/Melbourne.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Australia/Melbourne.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Australia/Melbourne.ics	2001-10-30 13:45:27.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Australia/Melbourne.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,15 +2,8 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Australia/Melbourne
+TZID:/softwarestudio.org/Olson_20070227_6/Australia/Melbourne
 X-LIC-LOCATION:Australia/Melbourne
-BEGIN:STANDARD
-TZOFFSETFROM:+1100
-TZOFFSETTO:+1000
-TZNAME:EST
-DTSTART:19700329T030000
-RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:+1000
 TZOFFSETTO:+1100
@@ -18,5 +11,12 @@
 DTSTART:19701025T020000
 RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:+1000
+TZOFFSETTO:+1000
+TZNAME:EST
+DTSTART:19700329T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Australia/Perth.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Australia/Perth.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Australia/Perth.ics	2001-10-30 13:45:27.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Australia/Perth.ics	2007-04-03 18:55:57.000000000 +0200
@@ -5,10 +5,16 @@
 TZID:/softwarestudio.org/Olson_20011030_5/Australia/Perth
 X-LIC-LOCATION:Australia/Perth
 BEGIN:STANDARD
-TZOFFSETFROM:+0800
+DTSTART:16010101T030000
+TZOFFSETFROM:+0900
 TZOFFSETTO:+0800
-TZNAME:WST
-DTSTART:19700101T000000
+RRULE:FREQ=YEARLY;INTERVAL=1;BYDAY=-1SU;BYMONTH=3
 END:STANDARD
+BEGIN:DAYLIGHT
+DTSTART:16010101T020000
+TZOFFSETFROM:+0800
+TZOFFSETTO:+0900
+RRULE:FREQ=YEARLY;INTERVAL=1;BYDAY=-1SU;BYMONTH=10
+END:DAYLIGHT
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Australia/Sydney.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Australia/Sydney.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Australia/Sydney.ics	2001-10-30 13:45:28.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Australia/Sydney.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,15 +2,8 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Australia/Sydney
+TZID:/softwarestudio.org/Olson_20070227_6/Australia/Sydney
 X-LIC-LOCATION:Australia/Sydney
-BEGIN:STANDARD
-TZOFFSETFROM:+1100
-TZOFFSETTO:+1000
-TZNAME:EST
-DTSTART:19700329T030000
-RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
-END:STANDARD
 BEGIN:DAYLIGHT
 TZOFFSETFROM:+1000
 TZOFFSETTO:+1100
@@ -18,5 +11,12 @@
 DTSTART:19701025T020000
 RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
 END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:+1000
+TZOFFSETTO:+1000
+TZNAME:EST
+DTSTART:19700329T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
+END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Europe/Istanbul.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Europe/Istanbul.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Europe/Istanbul.ics	2001-10-30 13:45:28.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Europe/Istanbul.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,20 +2,20 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Europe/Istanbul
+TZID:/softwarestudio.org/Olson_20070227_6/Europe/Istanbul
 X-LIC-LOCATION:Europe/Istanbul
 BEGIN:DAYLIGHT
 TZOFFSETFROM:+0200
 TZOFFSETTO:+0300
 TZNAME:EEST
-DTSTART:19700329T030000
+DTSTART:19700329T010000
 RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
 END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:+0300
 TZOFFSETTO:+0200
 TZNAME:EET
-DTSTART:19701025T040000
+DTSTART:19701025T020000
 RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
 END:STANDARD
 END:VTIMEZONE
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Europe/Vilnius.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Europe/Vilnius.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Europe/Vilnius.ics	2001-10-30 13:45:28.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Europe/Vilnius.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,13 +2,21 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_5/Europe/Vilnius
+TZID:/softwarestudio.org/Olson_20070227_6/Europe/Vilnius
 X-LIC-LOCATION:Europe/Vilnius
-BEGIN:STANDARD
+BEGIN:DAYLIGHT
 TZOFFSETFROM:+0200
+TZOFFSETTO:+0300
+TZNAME:EEST
+DTSTART:19700329T030000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
+END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:+0300
 TZOFFSETTO:+0200
 TZNAME:EET
-DTSTART:19700101T000000
+DTSTART:19701025T040000
+RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Pacific/Chatham.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Pacific/Chatham.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Pacific/Chatham.ics	2001-10-30 13:45:30.000000000 +0100
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Pacific/Chatham.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,15 +2,8 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20011030_4/Pacific/Chatham
+TZID:/softwarestudio.org/Olson_20070227_5/Pacific/Chatham
 X-LIC-LOCATION:Pacific/Chatham
-BEGIN:DAYLIGHT
-TZOFFSETFROM:+1245
-TZOFFSETTO:+1345
-TZNAME:CHADT
-DTSTART:19701004T024500
-RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=1SU
-END:DAYLIGHT
 BEGIN:STANDARD
 TZOFFSETFROM:+1345
 TZOFFSETTO:+1245
@@ -18,5 +11,12 @@
 DTSTART:19700315T034500
 RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=3SU
 END:STANDARD
+BEGIN:DAYLIGHT
+TZOFFSETFROM:+1245
+TZOFFSETTO:+1345
+TZNAME:CHADT
+DTSTART:19701004T024500
+RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=1SU
+END:DAYLIGHT
 END:VTIMEZONE
 END:VCALENDAR
diff -Nur evolution-data-server-1.8.2/calendar/libical/zoneinfo/Pacific/Tongatapu.ics evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Pacific/Tongatapu.ics
--- evolution-data-server-1.8.2/calendar/libical/zoneinfo/Pacific/Tongatapu.ics	2002-07-26 15:52:41.000000000 +0200
+++ evolution-data-server-1.8.2.new/calendar/libical/zoneinfo/Pacific/Tongatapu.ics	2007-04-03 18:55:57.000000000 +0200
@@ -2,21 +2,13 @@
 PRODID:-//Ximian//NONSGML Evolution Olson-VTIMEZONE Converter//EN
 VERSION:2.0
 BEGIN:VTIMEZONE
-TZID:/softwarestudio.org/Olson_20020614_6/Pacific/Tongatapu
+TZID:/softwarestudio.org/Olson_20070227_7/Pacific/Tongatapu
 X-LIC-LOCATION:Pacific/Tongatapu
-BEGIN:DAYLIGHT
-TZOFFSETFROM:+1300
-TZOFFSETTO:+1400
-TZNAME:TOST
-DTSTART:19701101T020000
-RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
-END:DAYLIGHT
 BEGIN:STANDARD
-TZOFFSETFROM:+1400
+TZOFFSETFROM:+1300
 TZOFFSETTO:+1300
 TZNAME:TOT
-DTSTART:19700125T020000
-RRULE:FREQ=YEARLY;BYMONTH=1;BYDAY=-1SU
+DTSTART:19700101T000000
 END:STANDARD
 END:VTIMEZONE
 END:VCALENDAR
